index.bs

```diff
@@ -1008,10 +1007,12 @@ Individual Participation Criteria</h3>
 	“Disclosure”
 	in the W3C Patent Policy [[!PATENT-POLICY]].
 
-	The [=Director=] <em class="rfc2119">may</em> suspend or remove for cause
-	a participant in any group (including the [=AB=] and [=TAG=]),
-	where cause includes failure to meet the requirements of this process,
-	the membership agreement, or applicable laws.
+	When serious and/or repeated violations do occur,
+	including failure to meet the requirements of this process,
```

Comment:
<ins>such as</ins><del>including</del> failure to meet the requirements of this process<del>,</del>

Reply:
```suggestion
	such as failure to meet the requirements of this process,
```